app/js/views/reminders.jsx

```diff
@@ -101,6 +105,70 @@ export default class Reminders extends React.Component {
       });
   }
 
+  /**
+   * Generate a phrase to be spoken to confirm a reminder.
+   *
+   * @param {Object} reminder
+   * @returns {string}
+   */
+  confirmAReminder(reminder) {
```

Comment:
This is part of the view for now and I agree it's pretty messy. In the future, this will probably be moved to the intent parser because 1. we need more context about the reminder and 2. some code can be shared.